src/snyk/extension.ts

```diff
@@ -264,7 +269,7 @@ class SnykExtension extends SnykLib implements IExtension {
     this.ossService.activateSuggestionProvider();
     this.ossService.activateManifestFileWatcher(this);
 
-    void this.notificationService.init();
+    void (await this.notificationService.init());
```

Comment:
You don't need await here, as we're not interested in return result of the function, nor do any handling with it.